pkg/controller/controllers.go

```diff
@@ -8,19 +8,25 @@ import (
 
 	"github.com/jackc/pgx/v4/pgxpool"
 	policiesv1 "github.com/open-cluster-management/governance-policy-propagator/api/v1"
+	chanv1 "github.com/open-cluster-management/multicloud-operators-channel/pkg/apis/apps/v1"
 	appsv1 "github.com/open-cluster-management/multicloud-operators-placementrule/pkg/apis/apps/v1"
+	subv1 "github.com/open-cluster-management/multicloud-operators-subscription/pkg/apis/apps/v1"
 	configv1 "github.com/stolostron/hub-of-hubs-data-types/apis/config/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	applicationv1 "sigs.k8s.io/application/api/v1beta1"
```

Comment:
let's try to use readable names as alias.
channlesv1, subscriptionsv1, appsv1beta1
the current alias appsv1 should be placementrulesv1